### PR TITLE
LPS-179163 [Site Initializer] Add Site Memberships to Users

### PR DESCRIPTION
**Pull request by nNilton**

### CODE REVIEW

**Comment by nNilton** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/user-accounts.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+[
+	{
+		"accountBriefs": [
+			{
+				"externalReferenceCode": "TESTACC0001"
+			}
+		],
+		"alternateName": "testAlternateName1",
+		"emailAddress": "test.user1@liferay.com",
+		"externalReferenceCode": "USER-1",
```
Comment (joseabelenda) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/user-accounts.json:10]: Should it be TESTUSER1?

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/user-accounts.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		"organizationBriefs": [
+			{
+				"name": "Test Organization 1"
+			}
+		]
+	},
+	{
+		"accountBriefs": [
+			{
+				"externalReferenceCode": "TESTACC0002"
```
Comment (joseabelenda) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-2/src/main/resources/site-initializer/user-accounts.json:23]: Remove zeros.

**Review by joseabelenda — CHANGES_REQUESTED** [review]

**Comment by joseabelenda** [comment]
resent:https://github.com/pt-liferay-solutions/liferay-portal/pull/3707

**Pull request closed without merging**